### ISSUE TITLE
Parsing error: No Babel config file detected for [GJS files]

### ISSUE THREAD

**Issue by Techn1x** [NullVoxPopuli/ember-eslint-parser#31]
Strangely, I get this error only for GJS files 🤔 
```
Parsing error: No Babel config file detected for /tests/integration/components/search-box-component-test.gjs. Either disable config file checking with requireConfigFile: false, or configure Babel so that it can find the config files
```

My eslint config looks like this
```js
module.exports = {
  overrides: [
    // JS / TS
    {
      files: ['**/*.{js,ts}'],
      plugins: ['ember', '@typescript-eslint'],
      parser: '@typescript-eslint/parser',
      extends: [
        'eslint:recommended',
        'plugin:ember/recommended',
        'plugin:@typescript-eslint/recommended',
        // From eslint-config-prettier - this disables stylistic rules that prettier handles. should be last.
        'prettier',
      ],
      rules: {
        ...commonTypescriptRules,
        ...commonEmberRules,
      },
    },
    // GTS
    {
      files: ['**/*.gts'],
      plugins: ['ember'],
      parser: 'ember-eslint-parser',
      extends: [
        'eslint:recommended',
        'plugin:ember/recommended',
        'plugin:ember/recommended-gts',
        // From eslint-config-prettier - this disables stylistic rules that prettier handles. should be last.
        'prettier',
      ],
      rules: {
        ...commonTypescriptRules,
        ...commonEmberRules,
      },
      globals: {
        // give glint/gts files access to global types
        ModelFor: 'readonly',
        Nullable: 'readonly',
        StudentEventsService: 'readonly',
      },
    },
    // GJS
    {
      files: ['**/*.gjs'],
      plugins: ['ember'],
      parser: 'ember-eslint-parser',
      extends: [
        'eslint:recommended',
        'plugin:ember/recommended',
        'plugin:ember/recommended-gjs',
        // From eslint-config-prettier - this disables stylistic rules that prettier handles. should be last.
        'prettier',
      ],
      rules: {
        ...commonEmberRules,
      },
    },
```

I can add an additional rule to make it the error go away
```
    {
      files: ['**/*.gjs'],
      parserOptions: {
        requireConfigFile: false,
      },
    },
```

But in doing so I end up with a different error in GJS files that use decorators

```
Parsing error: This experimental syntax requires enabling one of the following parser plugin(s): "decorators", "decorators-legacy". (48:6)
```

I am wondering if there is something that was configured for GTS but not GJS? Or is there some issue with my config?

**Comment by patricklx** [comment]
Maybe a rule? Prettier?

**Comment by NullVoxPopuli** [comment]
Looks like reproduction is easy: https://github.com/NullVoxPopuli/ember-eslint-parser/pull/32

@patricklx that is something we need to configure in the recommended config docs for users, yeah?

**Comment by Techn1x** [comment]
Let me know if I can help here in any way. Happy to try config's and test theories.

**Comment by patricklx** [comment]
fixed by #32

**Comment by Techn1x** [comment]
Can confirm this works great now in my project. Thanks!
